src/rest/rest-api.php

```diff
@@ -55,6 +55,13 @@ public static function permission_callback() {
 	}
 
 	public static function get_block_content( $params ) {
+		if ( isset( $params[ 'skip_blocks' ] ) && isset( $params[ 'include_blocks' ] ) ) {
+			return new WP_Error( 'vip-block-data-api-invalid-params', 'Cannot use both skip_blocks and include_blocks parameters', [ 'status' => 400 ] );
```

Comment:
`$content_parser->parse()` is also capable of returning `WP_Error` results (like `vip-block-data-api-no-blocks`). It might make sense to move this return into the `ContentParser` class so that an error is returned even if it's called directly.

Reply:
Yip, agreed. Done